productivity-suite/app/legacy-app/src/components/pages/News.tsx

```diff
@@ -45,9 +24,11 @@ export function News() {
 
   useEffect(() => {
     return getBus().listen<{ page: number }>("change-news-page", ({ page }) => {
-      setSearchParams({
-        page: `${page}`,
-      });
+      if (page) {
```

Comment:
Avoid setting the search params if there is not a valid page.
Otherwise you see things like:  `/calendar?page=undefined`

Reply:
When we cleanup the fragment we need a way to specify that there is no longer any "news page". I don't believe that setting it to 1 when we have left the fragment is a good idea. So we must "wipe out" the message by dispatching `undefined` or `{page: undefined}`. Either way we would then need to handle the undefined case.

Reply:
ah, I see 🙂👍